src/util/pg.py

```diff
@@ -362,14 +362,26 @@ def explode_execute(cr, query, table, alias=None, bucket_size=10000, logger=_log
     )
 
 
+class SQLStr(sql.SQL, str):
```

Comment:
I've been thinking about this, and do we really need to make this `sql.SQL`?
I'd propose we use a similar approach as `util.PGRegexp`
```suggestion
class SQLStr(str):
    pass
```
Then we check in `util.format_query` if the parameter is of this type, in which case we use literally (same as `sql.Composable`).

Reply:
Actually, this is a good idea.